build-system/tasks/serve.js

```diff
@@ -59,7 +59,17 @@ function serve() {
       'SERVE_CACHING_HEADERS': sendCachingHeaders,
     },
     stdout: !quiet,
-  }).once('quit', function() {
+  };
+
+  if (argv.debug) {
```

Comment:
For clarity, rename this flag to `inspect`, since that's what it's doing.

Reply:
Done.